docs/src/main/asciidoc/elytron-properties-guide.adoc

```diff
@@ -0,0 +1,168 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Elytron Properties File Config
+
+include::./attributes.adoc[]
+
+Quarkus provides support for properties file based authentication that is intended for
+development and testing purposes. It is not recommended that this be used in production as at present only
+plaintext passwords are supported.
+
+Add the following to your `pom.xml`:
+
+[source,xml]
+----
+<dependencies>
+    <!-- Elytron Security extension -->
+    <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-elytron-security-properties-file</artifactId>
+    </dependency>
+</dependencies>
+----
+
+== Configuration
+
+The elytron-security-properties-file extension currently supports two different realms for the storage of authentication
+and authorization information. Both support storage of this information in properties type files. The next two sections detail the specific configuration properties.
+
+=== Property Files Realm Configuration
+
+The property files realm supports mapping of users to password and users to roles with a combination of properties files. To enable and configure it, the following configuration properties are used:
+
+|===
+|Property Name|Default|Description
+|quarkus.users.file.enabled|false|Determine whether security via the file realm is enabled
```

Comment:
I wonder if it would be better to have a `security` prefix (so `quarkus.security.users`). For users, it might be self explanatory but it's not a given it will be for everything.